content/source/docs/enterprise/admin/backup-restore.html.md

```diff
@@ -71,14 +92,20 @@ Status  | Response           | Reason
 
```

Comment:
The PR as it stands looks good to me. One thing that I think may be of value to add is that in order for the backup to be successful, users *must* receive a `200 OK` back at the end of the call. I've seen cases where the backup timed out, no `200` was received, and the subsequent restore was missing data (as the backup had not completed successfully). 

Perhaps something like this?

```suggestion
~> **Important:** In order for a backup and restore to be successful, the backup API **must** return `200`. If `200` is not returned and the call silently closes, the backup blob may be incomplete, and may result in data loss. 
```

Reply:
Done!